test/helpers/test.go

```diff
@@ -586,6 +615,12 @@ func (t *TestOptions) WaitForFirstRelease() bool {
 	return strings.ToLower(text) != "true"
 }
 
+// TestChatOpsCommands should we test prow ChatOps commands
+func (t *TestOptions) TestChatOpsCommands() bool {
```

Comment:
I feel like this would be better named as `ShouldTestChatOpsCommand` - naming anything starting with `Test` that isn't an actual test makes me nervous. =)

Reply:
That's a great point. Didn't even think of that.